README.md

```diff
@@ -170,3 +175,26 @@ host machine isn't running Linux. If you are using a Mac, you'll need to run
 
     yarn install --frozen-lockfile
     npm run-script flow
+
+
+#### 6) Integration with Micromasters
```

Comment:
Can you change `Micromasters` to `MicroMasters` here and below?

Reply:
✅